cts/cts-scheduler.in

```diff
@@ -242,6 +242,7 @@ TESTS = [
         [ "coloc-dependee-should-move", "Group colocation outweighs stickiness" ],
         [ "colocation-influence", "Respect colocation influence" ],
         [ "colocation-priority-group", "Apply group colocations in order of primary priority" ],
+        [ "colocation-vs-stickiness", "Group stickiness outweighs anti-colocation score" ],
```

Comment:
The test description isn't intended to mean that stickiness always outweighs anti-colocations, but that stickiness should win with this particular input because its score is higher.

Clarifying the docs is a good idea